conf/rmwork.conf

```diff
@@ -1,2 +1,18 @@
 RMWORK ?= "0"
 RMWORK[nohash] = True
+
+# By default, remove everything but ${WORKDIR}/tmp aka ${T}. We refuse
+# to remove a directory which is not a subdirectory of ${WORKDIR}.
+RMWORK_DIRS ?= """
+${STAGE_DIR}
+${STAGE_UNPACKDIR}
+${RSTAGE_DIR}
+${FSTAGE_DIR}
+${SRCDIR}
+${S}
+${B}
+${D}
+${PKGD}
+${IMAGE_DIR}
+"""
```

Comment:
Well, I think that's what's done up until the last patch, but I thought it might be nice to be able to control it. If one would only use it for saying `RMWORK_DIRS = ""` to disable it for a specific recipe, there's of course a simpler way to do that, but this didn't seem to be much more work. I have no strong feelings either way.